src/exchange/ExchangeHomeScreen.tsx

```diff
@@ -150,7 +157,17 @@ function ExchangeHomeScreen() {
             </View>
           </View>
 
-          <CeloGoldHistoryChart testID="PriceChart" />
+          {usePriceHistoryFromBlockchainApi && tokensBySymbol['CELO']?.tokenId ? (
```

Comment:
nit: can we just use `networkConfig.celoTokenId` instead of extracting this from tokensBySymbol?

Reply:
Fixed in c138a17891cf98987f197d691a48ea1e5890a04e!